website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx

```diff
@@ -0,0 +1,147 @@
+---
+page_title: Audit Trails Tokens - API Docs - HCP Terraform
+description: >-
+  Use the `/authentication-token` endpoint with the `token` query param to manage audit trails API tokens. Generate and delete audit trails tokens using the HTTP API.
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+[JSON API document]: /terraform/cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+# Audit Trails Token API
+
+## Generate a new audit trails token
+
+`POST /organizations/:organization_name/authentication-token?token=audit-trails`
```

Comment:
I'm wondering if we need a new page here or could we just add `token` as a parameter to [https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organization-tokens](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organization-tokens)

Reply:
If we have separate pages for every type of org token then I think we should follow that pattern for consistency. I'm totally open to continuing this conversation if you disagree though!

Reply:
Ok... If you're happy I'm happy :-) .

I don't know how many pages we'll end up with in the future, but I pretty confident it's not going to be in the 100s or even 10s of token types. So the navbar should be ok.